ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-ref.metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+	"path": "ipi/conf/aws/outpost/ipi-conf-aws-outpost-ref.yaml",
+	"owners": {
+		"approvers": [
+			"wking"
```

Comment:
wking is not responsible for installation, add you and your team member here.

Reply:
How do I get my name?  is it 'AsherShoan' ?